Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -2443,6 +2448,15 @@ - (void)handleAppState
             //  wait for another session state change to check room list data is ready
             return;
         }
+
+        if (mainSession.vc_homeserverConfiguration.encryption.isSecureBackupRequired
```

Comment:
I think we should get https://github.com/vector-im/element-ios/pull/5476 merged also as there was an issue that could happen around this screen being inadvertently displayed in some cases.